__tests__/api/utils.test.js

```diff
@@ -32,44 +49,77 @@ describe('Utils', () => {
     });
   });
 
+  describe('createReverseRelationships()', () => {
+    const state = {
+      resources: {
+        people: {
+          9: {
+            relationships: {
+              comments: {
+                data: [
+                  { id: '12', type: 'comments' }
+                ]
+              }
+            }
+          }
+        },
+        comments: {
+          5: {},
+          12: {}
+        },
+      }
+    };
+
+    test('updates hasMany relationships', () => {
+      const newState = imm(state);
+      utils.createReverseRelationships(state, newState, comments[0]);
```

Comment:
🤔 I think we have multiple test files currently depending on the json-api-body fixture.